infra/modules/cloudfunctions/metadata.yaml

```diff
@@ -24,7 +24,7 @@ spec:
     source:
       repo: https://github.com/GoogleCloudPlatform/terraform-ml-image-annotation-gcf.git
       sourceType: git
-      dir: /infra/modules/cloudfunctions
+      dir: cloudfunctions
```

Comment:
If relative to repo root, shouldn't "/infra/modules/cloudfunctions" be correct?

Reply:
All changes to the metadata.yaml and metadata.display.yaml were auto-generated when I ran `make docker_generate_docs`. ATM these files are not used anywhere. But we are working with the blueprints team to improve the process. So these changes can be ignored.